src/components/TestDetailsDialog/TestDetailsModal.tsx

```diff
@@ -166,54 +174,58 @@ const TestDetailsModal: React.FunctionComponent<{
     setApplyIgnoreDialogOpen(!applyIgnoreDialogOpen);
   };
 
-  console.log("rerender TestDetailsModal");
-  const [image, imageStatus] = useImage(
+  const [currentImage, currentImageStatus] = useImage(
     staticService.getImage(testRun.imageName),
   );
+
   const [baselineImage, baselineImageStatus] = useImage(
     staticService.getImage(testRun.baselineName),
   );
+
   const [diffImage, diffImageStatus] = useImage(
     staticService.getImage(testRun.diffName),
   );
 
   const applyIgnoreAreaText =
     "Apply selected ignore area to all images in this build.";
 
-  const fitImageToStage = (image: HTMLImageElement, container: HTMLElement) => {
+  const fitImageToStage = (img: HTMLImageElement, container: HTMLElement) => {
```

Comment:
could you revert refactoring changes that are not related to library migration?
we could make it later not to mix different changes